static/build.html

```diff
@@ -656,21 +656,18 @@ <h3 id="browser-and-webview">
             Chromium's Android build instructions</a> for details on obtaining the
             prerequisites.</p>
 
-            <p>You can obtain the proper configuration from the
-            <a href="https://github.com/GrapheneOS/Vanadium">
-            GrapheneOS Vanadium repository</a> in <code>args.gn</code> including the correct
-            version.</p>
-
             <pre>git clone https://github.com/GrapheneOS/Vanadium.git
-cd Vanadium
-git checkout $CORRECT_BRANCH_OR_TAG</pre>
+            cd Vanadium
```

Comment:
This whitespace shouldn't be added since the pre tag will include it.

Reply:
Update made. I really thought I could squash commits on the site, I will try to make less noise with changes and work locally before pushing commits.